src/collections/_documentation/api/events/delete-project-group-index.md

```diff
@@ -1,12 +1,12 @@
 ---
-# This file is automatically generated from the API using `api-docs/generate.py`
-# Do not manually this file.
+# This file is automatically generated from the API using `sentry/api-docs/generator.py.`
+# Do not manually edit this file.
 {
   "api_path": "/api/0/projects/{organization_slug}/{project_slug}/issues/", 
   "authentication": "required", 
   "description": "Permanently remove the given issues. The list of issues to\nmodify is given through the `id` query parameter.  It is repeated\nfor each issue that should be removed.\n\nOnly queries by 'id' are accepted.\n\nIf any ids are out of scope this operation will succeed without\nany data mutation.", 
-  "example_request": "DELETE /api/0/projects/the-interstellar-jurisdiction/amazing-plumbing/issues/?id=5&id=6 HTTP/1.1\nHost: sentry.io\nAuthorization: Bearer <token>", 
-  "example_response": "HTTP/1.1 204 NO CONTENT\nContent-Length: 0\nX-XSS-Protection: 1; mode=block\nContent-Language: en\nX-Content-Type-Options: nosniff\nVary: Accept-Language, Cookie\nAllow: GET, PUT, DELETE, HEAD, OPTIONS\nX-Frame-Options: deny", 
+  "example_request": "", 
+  "example_response": "", 
```

Comment:
What happened here?

Reply:
Was removed here: https://github.com/getsentry/sentry/pull/11623/files#diff-25a63c81785c41e73ddc87f4058a70a4L800. Looks accidental. Going to add it back.

Reply:
Added. https://github.com/getsentry/sentry/pull/16911/commits/d1f46fb298d23c541df8cc8a8703297e8d355256